sal_interfaces/ATWhiteLight/ATWhiteLight_Events.xml

```diff
@@ -1,219 +1,332 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
 <SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
-  <Enumeration>DetailedState_DisabledState,DetailedState_EnabledState,DetailedState_FaultState,DetailedState_OfflineState,DetailedState_StandbyState</Enumeration>
+  <Enumeration>
+    ChillerL1Alarms_AMBIENT_TEMP_SENSOR = 0x01,
+    ChillerL1Alarms_HIGH_CONTROL_TEMP = 0x02,
+    ChillerL1Alarms_PT7_HIGH_TEMP = 0x04,
+    ChillerL1Alarms_LOW_CONTROL_TEMP = 0x08,
+
```

Comment:
Why all these single and double empty lines in the enums? If possible please remove them.

Reply:
I was grouping by 4s and indicating missing values. I'll remove the grouping extra lines, leaving only an extra line where one or more values is intentionally missing (because the vendor isn't using it).